### ISSUE TITLE
Error: Wrapper command 'wine64' not found on the system even after installing wine-stable using Homebrew

### ISSUE THREAD

**Issue by mkapal** [astracompiler/cli#64]
Terminal output:

```
❯ astra build ./dist/index.js                                                                                   

  Astra v1.2.0
  You're a coding beast!

Tips:
1. Architecture of your system is arm64
2. Your node version is v20.19.2
3. If you don't know which version to choose, select the latest LTS version or type node -v in your terminal

✔ Select a version of your project v22.15.1 win x64 LTS
ℹ  info      Version already installed.
▶  start     [1/4] Building project...
✔  success   Project built!
▶  start     [2/4] Setting file metadata...
✖  fatal     Error: Wrapper command 'wine64' not found on the system. Run `brew install --cask wine-stable` to install 64-bit wine on macOS via Homebrew.

❯ brew install --cask wine-stable                                                                         
Warning: Not upgrading wine-stable, the latest version is already installed
```

**Comment by mkapal** [comment]
I've just noticed a closed issue related to this: https://github.com/astracompiler/cli/issues/1

However I think it should not show the error anymore after it's been installed via homebrew.
